src/ai-bundle/doc/index.rst

```diff
@@ -91,6 +91,15 @@ Configuration
                 # multiple collections possible per type
                 default:
                     collection: 'my_collection'
+            cache:
+                cache:
+                    research:
+                        service: 'cache.app'
+                        cache_key: 'research'
+                        strategy: 'chebyshev'
+                memory:
+                    ollama:
+                        strategy: 'chebyshev'
```

Comment:
if i get it right, this should be:
```suggestion
            cache:
                research:
                    service: 'cache.app'
                    cache_key: 'research'
                    strategy: 'chebyshev'
            memory:
                in_memory:
                    strategy: 'chebyshev'
```

Reply:
Of course, I lost myself in the spaces 😄